src/components/ActionButton/ActionButton.js

```diff
@@ -21,14 +22,18 @@ const ActionButton = ({
   const Element = tag || 'button';
   return (
     <Element
-      onClick={onClick}
+      onClick={(e) => {
```

Comment:
```suggestion
      onClick={disabled ? undefined : onClick }
```

will avoid re-render :)

Reply:
I like it!